src/functions/lap.js

```diff
@@ -44,19 +44,16 @@ defineFunction({
         // two items involved in the lap.
         // Next, use a strut to set the height of the HTML bounding box.
         // Otherwise, a tall argument may be misplaced.
+        // This code resolved issue #1153
         const strut = buildCommon.makeSpan(["strut"]);
         strut.style.height = (node.height + node.depth) + "em";
         strut.style.verticalAlign = -node.depth + "em";
         node.children.unshift(strut);
 
         // Next, prevent vertical misplacement when next to something tall.
-        node = buildCommon.makeVList({
-            positionType: "firstBaseline",
-            children: [{type: "elem", elem: node}],
-        }, options);
-
-        // Get the horizontal spacing correct relative to adjacent items.
-        return buildCommon.makeSpan(["mord"], [node], options);
+        // This code resolves issue #1234
+        node = buildCommon.makeSpan(["thinbox"], [node], options);
+        return buildCommon.makeSpan(["mord", "vbox"], [node], options);
```

Comment:
Are there other commands we can simplify in similar ways?

Reply:
The big opportunity is in `makeVList()`. It writes a lot of HTML and you demonstrated in issue #1409 that it could be made much less verbose by use of `flex`.

Reply:
I totally forgot about that, thanks for the reminder. 😅